subprojects/resources-gcs/build.gradle.kts

```diff
@@ -36,8 +36,3 @@ dependencies {
 strictCompile {
     ignoreDeprecations()
 }
-
```

Comment:
The changes here were enough to make the resources-gcs tests pass, as the ProgressLoggingFixture service leakage was the only thing in the way.